### PR TITLE
feat(sign-off): Map signoff and add edit signoff for item(SDNTB-24, S…

### PR DESCRIPTION
**Pull request by ioanpocol**
…D-NTB-25)

Those changes are available only if sign_off mapping is set on
settings.py and index.html
When an user is added/edited, the sign-off is not shown and is set to
mapped value
On authoring the user is able to set a new value for sign-off by
selecting only one user from user list

### CODE REVIEW

File: scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -2330,6 +2333,22 @@
                 };
 
                 /**
+                 * Modify the sign-off with the value from sign_off_mapping field from user
+                 */
+                scope.modifySingOff = function(user) {
```
Comment (akintolga) [inline scripts/superdesk-authoring/authoring.js]: typo in the name of the method...

**Comment by akintolga** [comment]
:+1:

File: scripts/superdesk-authoring/views/article-edit.html [hunk 2]
```diff
@@ -201,5 +201,11 @@
 
 <div class="field pull-left" ng-if="item.sign_off && schema.sign_off" order="{{schema.sign_off.order}}">
     <label translate>Sign-Off</label>
-    <div>{{item.sign_off}}</div>
+    <div>{{item.sign_off}}
+      <button class="btn btn-signOff" ng-click="changeSignOffEdit()" title="Unlock edit Sign-Off" ng-if="canEditSignOff && _editable">
+  		  <i class="icon-lock" ng-hide="editSignOff"></i>
```
Comment (pavlovicnemanja) [inline scripts/superdesk-authoring/views/article-edit.html]: Maybe you can add <i `ng-class="{'icon-lock': !editSignOff, 'icon-trash': editSignOff}"></i>`

**Comment by pavlovicnemanja** [comment]
:+1:

**Pull request merged**